src/filter.js

```diff
@@ -0,0 +1,276 @@
+// Change State hook of App to change the markers on the map
+
+import axios from "axios";
+
+let buttonValue = {
+    Distance: false,
+    Rating: false,
+    Name: false,
+    Partner: false,
+    Public: false,
+    Spring: false,
+    Cold: false,
+    Hot: false,
+    Filtered: false,
+    Tap: false,
+    Yourself: false,
+    Staff: false,
+    WiFi: false,
+    Toilet: false,
+    Accessible: false,
+    Space: false,
+}
+
+const buttonTags = {
+    Filtered: 1,
+    Hot: 2,
+    Cold: 3,
+    WiFi: 4,
+    Accessible: 5,
+    Tap: 6,
+}
+
+
+let position = {};
+let setPlaces;
+
+export function createFilter(map, maps, setTaps) {
+    const bounds = map.getBounds();
+    setPlaces = setTaps;
+    position = {
```

Comment:
Update the Bound's position each time the map is moved

Reply:
The function sets the bounds when the apply button is triggered ([094ae82](https://github.com/mymizu/mymizu-web/pull/21/commits/094ae827df9b0c1e320b039a6faf56ed635afadb))